viper.go

```diff
@@ -1123,6 +1123,12 @@ func (v *Viper) SetEnvKeyReplacer(r *strings.Replacer) {
 	v.envKeyReplacer = r
 }
 
+// Returns the alias map, used for documentation generation.
+func GetAliases() map[string]string { return v.GetAliases() }
+func (v *Viper) GetAliases() map[string]string {
+	return v.aliases
```

Comment:
Nit: Defensively speaking, a copy should be returned, instead of the internal data structure.

Reply:
Good point.  Is there a copy function I should use, or just iterate?